install.sh

```diff
@@ -230,7 +231,14 @@ resume_bamboo_server() {
       fi
       bamboo_url=$(terraform output | grep '"bamboo" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
       resume_bamboo_url="${bamboo_url}/rest/api/latest/server/resume"
-      curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}"
+      local RESULT=$(curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}")
+      if [[ "x${RESULT}" == *"RUNNING"* ]]; then
```

Comment:
Should we retry or do you want to rely on the delay?

Reply:
Delay works consistently, but yes, retry is a better approach. I will change this part to retry with a shorter interval and a longer expiry time.

Reply:
However, retry is a better approach in that case. We ca change that part to retry with a shorter interval and a longer expiry time in a new ticket.